overlord/install/install.go

```diff
@@ -81,13 +82,62 @@ type EncryptionSupportInfo struct {
 	UnavailableWarning string
 }
 
+// CompSeedInfo contains information for a component from the seed and
+// from its metadata.
+type CompSeedInfo struct {
```

Comment:
This is a nitpick but for me, when it comes to exported structures that are part of an API it feels more appropriate to spell it out `ComponentSeedInfo` instead of shorting it, which is what you might do for the variable name.

Reply:
Changed